notebooks/hello-universe/Regressing_TESS_rotation_periods_with_CNNs/Regressing_TESS_rotation_periods_with_CNNs.ipynb

```diff
@@ -0,0 +1,1019 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "true_filtered = true_periods[frac < 0.5]\n",
+    "pred_filtered = pred_periods[frac < 0.5]\n",
+    "plt.scatter(true_filtered, pred_filtered)\n",
+    "plt.plot([0, 180], [0, 180], \"k\")\n",
+    "plt.xlabel(\"True Period (days)\")\n",
+    "plt.ylabel(\"Predicted Period (days)\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "058fba43-4638-4967-a5f7-064d5488afaa",
+   "metadata": {},
+   "source": [
+    "With the \"bad\" predictions filtered out, the predictions look much better, even after only 20 epochs. Remember that the uncertainty is predicted from the quality of the data, so this kind of cut can be applied to predictions on real data as well. \n",
+    "\n",
+    "Finally, let's define an accuracy metric to measure the performance. For this example, we'll use the root-mean-squared (RMS) error."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "2d9cbdfe-44f3-4dae-9da3-caea46be9258",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Unfiltered prediction count: 1000\n",
+      "RMS error: 48.68 days\n",
+      "\n",
+      "Filtered prediction count:   123\n",
+      "RMS error: 31.30 days.\n"
+     ]
+    }
+   ],
+   "source": [
+    "def rms_error(true, pred):\n",
+    "    return np.sqrt(np.mean((true - pred)**2))\n",
+    "\n",
+    "print(f\"Unfiltered prediction count: {len(pred_periods)}\\n\"\n",
+    "      f\"RMS error: {rms_error(true_periods, pred_periods):.2f} days\\n\\n\"\n",
+    "      f\"Filtered prediction count:   {len(pred_filtered)}\\n\"\n",
+    "      f\"RMS error: {rms_error(true_filtered, pred_filtered):.2f} days.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "508fe865-deac-4253-b18d-5221616e277f",
+   "metadata": {},
+   "source": [
+    "When the predictions are filtered, we retain only about 10-15% of the sample, but the accuracy improves significantly. Some takeaway notes:\n",
```

Comment:
I think you need a bit more of a summary.  Your "takeaway notes" here are basically "what someone who already understands the general method might learn from this specific example", but what you really want is a summary for someone who doesn't really know what a CNN is or why this method might be useful.

This might be as simple as just something like adding a note "CNNs combined with wavelets make reasonable predictions for TESS light curves if trained on data with different systematics."  Which I think is your main point?  But if not, please decide what it is and write it here!